### PR TITLE
Updated yaml - Env variable groups

### PR DESCRIPTION
**Pull request by K-Codemagic**
Updated yaml - Env variable groups

### CODE REVIEW

**Review by icarusdust — APPROVED** [review]
LGTM

File: content/yaml/yaml-getting-started.md [hunk 1]
```diff
@@ -228,10 +238,9 @@ You can freely use all of the above features of environment section in conjuncti
 
 ```yaml
 environment:
-  vars: # Define your environment variables here
+  vars: # Define your public environment variables here
     PUBLIC_ENV_VAR: 'value here'
```
Comment (nevercode-kevin) [inline content/yaml/yaml-getting-started.md:242]: Can we use double quotes for consistency?

**Review by nevercode-kevin — CHANGES_REQUESTED** [review]

File: content/yaml/yaml-getting-started.md (hunk 1 above)
Comment (K-Codemagic) [reply content/yaml/yaml-getting-started.md]: suggested change has been made. Kindly approve it

**Review by nevercode-kevin — APPROVED** [review]
LGTM

**Pull request merged**